playwright/e2e/sliding-sync/sliding-sync.spec.ts

```diff
@@ -21,12 +21,14 @@ import type { ElementAppPage } from "../../pages/ElementAppPage";
 import type { Bot } from "../../pages/bot";
 
 test.describe("Sliding Sync", () => {
-    test.use({ usesSlidingSyncProxy: true });
+    test.use({
+        usesSlidingSyncProxy: true,
```

Comment:
`test use(s) uses sliding sync proxy` seems a bit duplicative, do we need the `uses` here?